bin/casher

```diff
@@ -71,11 +72,13 @@ class Casher
         break
       end
     end
-    unless archive_found
-      msg "could not download cache", :red
-      if File.exist? @fetch_tar
-        rm @fetch_tar
-      end
+    cleanup_cache unless archive_found
+  end
+
+  def cleanup_cache
+    msg "could not download cache", :red
+    if File.exist? @fetch_tar
+      rm @fetch_tar
```

Comment:
This isn't specific to the PR, but I don't know why we wouldn't use `FileUtils.rm_f(@fetch_tar)`.  Tell-don't-ask and all that jazz :saxophone: :man_shrugging: